src/webui/btjson.cpp

```diff
@@ -775,7 +775,8 @@ void processMap(QVariantMap prevData, QVariantMap data, QVariantMap &syncData)
                 syncData[key] = data[key];
             break;
         default:
-            Q_ASSERT(0);
+            Q_ASSERT_X(false, __FILE__ "@processMap"
```

Comment:
I think `"@processMap"` is enough. You don't need `__FILE__`. According to the [documentation](https://doc.qt.io/qt-5/qtglobal.html#Q_ASSERT_X) it will output source file and line number by itself anyway.